class.jetpack-connection-banner.php

```diff
@@ -230,7 +230,7 @@ class="notice-dismiss"
 
 					<!-- slide 1: intro -->
 					<div class="jp-wpcom-connect__slide jp-wpcom-connect__slide-one jp__slide-is-active">
-						<h2><?php esc_html_e( 'Jetpack is here to enhance your site and your life', 'jetpack' ) ?></h2>
+						<h2><?php esc_html_e( 'Jetpack simplifies site security, customization, and promotion.', 'jetpack' ) ?></h2>
```

Comment:
Promotion? What does that mean to an end user? It sounds like Jetpack is a marketing tool.